build.sbt

```diff
@@ -153,10 +150,28 @@ lazy val packagingSettings = Seq(
   mappings in (Compile, packageDoc) := Seq(),
   debianChangelog in Debian := Some(baseDirectory.value / "changelog.md"),
 
+  (packageName in Universal) := {
+    git.gitHeadCommit.value match {
+      case Some(commit) =>
+        val shortCommit = commit.take(7)
```

Comment:
why 7?

Reply:
It is a convention. Git SHAa are usually shortened to 7 chars. Though, I agree it would be nice to have it as a variable.

Reply:
Is there a strong reason to not get this variable from the version script?

Reply:
Not really. I wanted to keep the version script simple. I'll add a `version commit`.